README.md

```diff
@@ -115,7 +115,8 @@ app.use('/s3', require('react-s3-uploader/s3router')({
     region: 'us-east-1', //optional
     signatureVersion: 'v4', //optional (use for some amazon regions: frankfurt and others)
     headers: {'Access-Control-Allow-Origin': '*'}, // optional
-    ACL: 'private' // this is default
+    ACL: 'private', // this is default
+    prefixFilename=true // this is default, setting the attribute to false preserves the original filename in S3
```

Comment:
Can we rename this param to `uniquePrefix`?  I want convey the fact that this is for the purposes of making the filename unique, and not some arbitrary prefix.